android/build.gradle

```diff
@@ -15,6 +15,10 @@ android {
         ndk {
             abiFilters "x86", "armeabi-v7a", "x86_64", "arm64-v8a"
         }
+        compileOptions {
+            sourceCompatibility JavaVersion.VERSION_1_8
+            targetCompatibility JavaVersion.VERSION_1_8
+        }
```

Comment:
if that fixes the issue we can merge it but it should not be necessary https://developer.android.com/studio/releases/gradle-plugin#java-8-default

Reply:
@marandaneto it is by default in `4.2.0` version but most of the users are using `4.1.2` etc like me. therefore we should merge this PR to support this package for all users
i am using this version
```
classpath("com.android.tools.build:gradle:4.1.2")
```

Reply:
This works for me. Having an explicit Java version helps too. 
A lot of react-native projects are don't have `4.2.0` Android tools.

Reply:
@marandaneto what you say?

Reply:
yep, its the way to go, thanks